c/tskit/tables.c

```diff
@@ -23,6 +23,7 @@
  * SOFTWARE.
  */
 
+#include "tskit/core.h"
```

Comment:
This should be down in the other group? Unless there's something funky going on with shadowing the other imports?

Reply:
Yeah, I got stuck on this once I saw how the imports work out.  We don't need trees.h beyond a couple of constants.

Reply:
I was able to delete this w/no harm.  Not sure what I was thinking.